gitbootcamp.rst

```diff
@@ -77,18 +77,18 @@ Setting Up Your Name and Email Address
 
 .. code-block:: bash
 
-   git config --global user.name "Your Name"
-   git config --global user.email email@example.org
+   git config --global user.name <name>
+   git config --global user.email <email>
 
-The ``--global`` flag sets these globally,
-``--local`` sets them only for the current project.
+The ``--global`` flag sets these parameters globally while
+the ``--local`` flag sets them only for the current project.
```

Comment:
Since ``--global`` was explicitly mentioned as a flag earlier within the same sentence, it might be a bit redundant to do the same for ``--local``. I agree with the addition of "parameters" though.

Reply:
To me it the asymmetry sounded odd.